kairon/shared/actions/data_objects.py

```
@@ -769,6 +769,10 @@
         if self.top_results > 30:
             raise ValidationError("top_results should not be greater than 30")
 
+class CrudConfig(EmbeddedDocument):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)